tests/roots/test-outdir/conf.py

```diff
@@ -0,0 +1,12 @@
+import tempfile
+import os
+
+extensions = ["sphinxcontrib.drawio"]
+
+master_doc = "index"
+exclude_patterns = ["_build"]
+
+# removes most of the HTML
+html_theme = "basic"
+
+drawio_export_directory = os.path.abspath(tempfile.gettempdir() + "/drawio_outdir_test")
```

Comment:
Could we have a test on relative directory as well, since it's "supported".

Reply:
I will add a test